backend/index.test.js

```diff
@@ -27,27 +30,132 @@ describe("Server", () => {
     server.close();
   });
 
-  describe("GET /api/v1/estimates/:clientID", () => {
-    it("should return a 200 status code for a valid client ID", async () => {
-      const response = await supertest(server).get("/api/v1/estimates/63f82d40be153fa3c4b62062");
+  // router.route("/manage-profile").put(ProfileCtrl.apiUpdateProfile)
+  describe("PUT /api/v1/manage-profile", () => {
+    it("should return a 200 status code for valid json fields", async () => {
+      const response = await supertest(server).put("/api/v1/manage-profile").send({
+        "client_name": "Clara Martin",
+        "address_1" : "5500 Sampson",
+        "address_2" : "Apt. 5678",
+        "city" : "Houston",
+        "state" : "TX",
+        "zipcode" : "75555"
+      });
       expect(response.status).toBe(200);
     });
 
-    it("should return a JSON object with estimates and client_id keys", async () => {

```

Comment:
We'll need the estimates tests as well as yours for coverage, let's add those back before merging
Feel free to copy paste them from here (just make sure the brackets still work, since I'm pasting these in from the git diff):
```js
describe("GET /api/v1/estimates/:clientID", () => {
    it("should return a 200 status code for a valid client ID", async () => {
      const response = await supertest(server).get("/api/v1/estimates/63f82d40be153fa3c4b62062");
      expect(response.status).toBe(200);
    });
    it("should return a JSON object with estimates and client_id keys", async () => {
      const response = await supertest(server).get("/api/v1/estimates/63f82d40be153fa3c4b62062");
      expect(response.body).toEqual(
        expect.objectContaining({
          estimates: expect.any(Array),
          client_id: expect.any(String),
        })
      );
    });
    it("should return an empty array if no estimates are found for a client ID", async () => {
      const response = await supertest(server).get(
        "/api/v1/estimates/client_without_estimates"
      );
      expect(response.body.estimates).toEqual([]);
    });
  });
```

Reply:
Added!